docs/user-guide/io.md

```diff
@@ -167,32 +249,26 @@ For example, to set a stripe size of 4 MiB for the existing directory
 
     auser@ln03:~> lfs setstripe -S 4m -c -1 resdir/
 
-### Recommended ARCHER2 I/O settings
+### Environement variables
 
+These environment variables typically only have an impact for the case 
```

Comment:
"These"?

Reply:
I've changed some wording. Please could you have a look at it? @kevinstratford